.travis.yml

```diff
@@ -12,8 +12,6 @@ script:
   - pytest . --doctest-modules
       --ignore=data_structures/stacks/balanced_parentheses.py
       --ignore=data_structures/stacks/infix_to_postfix_conversion.py
-      --ignore=file_transfer_protocol/ftp_send_receive.py
```

Comment:
Was __ftp_send_receive.py__ fixed?

Reply:
I did not make any changes, but will test later